resources/assets/js/components/Application/MyFit/Question.tsx

```diff
@@ -0,0 +1,169 @@
+/* eslint-disable @typescript-eslint/camelcase */
+import * as React from "react";
+import { Formik, Form, FastField } from "formik";
+import * as Yup from "yup";
+import { useIntl, FormattedMessage, defineMessages } from "react-intl";
+import { JobPosterQuestion, JobApplicationAnswer } from "../../../models/types";
+import { getLocale, localizeField } from "../../../helpers/localize";
+import { validationMessages } from "../../Form/Messages";
+import { countNumberOfWords } from "../../WordCounter/helpers";
+import TextAreaInput from "../../Form/TextAreaInput";
+
+const saveButtonMessages = defineMessages({
+  save: {
+    id: "application.myfit.saveAnswerButton.default",
+    defaultMessage: "Save Answer",
+    description: "Default message displayed on save answer button.",
+  },
+  saved: {
+    id: "application.myfit.saveAnswerButton.saved",
+    defaultMessage: "Saved",
+    description: "Message displayed on button when data is saved.",
+  },
+});
+
+interface QuestionProps {
+  jobApplicationAnswer: JobApplicationAnswer;
+  formRef: any;
+  index: number;
+  question: JobPosterQuestion;
+  handleSubmit: (data: JobApplicationAnswer) => Promise<void>;
+}
+
+export interface QuestionValues {
+  id: number;
```

Comment:
If this is just the Form Values, I think the only property necessary here is the `answer` field.